coopr-server/src/main/java/co/cask/coopr/codec/json/current/LeaseDurationCodec.java

```diff
@@ -34,11 +35,44 @@ public LeaseDuration deserialize(JsonElement json, Type typeOfT, JsonDeserializa
     throws JsonParseException {
     JsonObject jsonObj = json.getAsJsonObject();
 
-    Long initial = context.deserialize(jsonObj.get("initial"), Long.class);
-    Long max = context.deserialize(jsonObj.get("max"), Long.class);
-    Long step = context.deserialize(jsonObj.get("step"), Long.class);
+    if (jsonObj.has("initial") && jsonObj.get("initial").isJsonPrimitive() &&
```

Comment:
though it is possible they are null if they were not specified in the json.  If they are not there, I think it makes sense to default the values to 0, which could be done in the builder.